### PR TITLE
Restyle the Facebook app channel claim pages

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
RapidPro - Visually build interactive SMS Applications 2020-08-25 13-21-08
RapidPro - Visually build interactive SMS Applications 2020-08-25 13-20-21
RapidPro - Visually build interactive SMS Applications 2020-08-25 13-20-07
RapidPro - Visually build interactive SMS Applications 2020-08-25 13-19-50

**Comment by ericnewcomer** [comment]
Thanks @norkans7 

Can we treat this more like a wizard? ie, lets not have a list of steps at the beginning. Instead let's hold their hand through each step.

It should start right away with the login/select user step with language like, "Which Facebook account do you use to manage your page?", then after that, they see the pages "Select the page you'd like to connect". And so on.

**Comment by norkans7** [comment]
We do not control the facebook login window that popup and that were all those steps need to happen

**Comment by rowanseymour** [comment]
@norkans7 can you figure out what's going on with the diff here...

File: templates/channels/types/facebookapp/claim.haml [hunk 1]
```diff
@@ -1,28 +1,46 @@
 -extends "channels/channel_claim_form.html"
 -load i18n compress temba
 
--block modal
+-block pre-form
+    -blocktrans trimmed with name=brand.name
```
Comment (rowanseymour) [inline templates/channels/types/facebookapp/claim.haml:5]: extra indentation

File: templates/channels/types/facebookapp/refresh_token.haml [hunk 2]
```diff
@@ -1,38 +1,29 @@
...
-      - trans "Error with token, you need to reconnect the Facebook page by clicking the button below"
-    -else
-      - trans "Everything looks good. No need to reconnect"
-  
-  
-  #fb-channel-error(style="display:none;")
-    %p
-      -trans "Error reconnecting Facebook page. Please retry"
+-block pre-form
+    %p#fb-status
```
Comment (rowanseymour) [inline templates/channels/types/facebookapp/refresh_token.haml:5]: extra indentation

**Comment by rowanseymour** [comment]
@ericnewcomer thoughts on whether this belongs in 6.0 as is and we convert to wizard later?

**Comment by ericnewcomer** [comment]
To @norkans7's point, the directions are the way they are because they are directing actions off site. I don't know enough about this facebook path, and honestly have not looked into it yet. I'm okay with it being a step forward for 6.0 and circling back though.

**Pull request merged**